frame/support/src/traits/misc.rs

```diff
@@ -407,6 +407,92 @@ where
 	}
 }
 
+/// Defensively checks the minimum between two values.
+pub trait DefensiveMin<T> {
+	/// Defensively checks the minimum between two values
```

Comment:
please wrap lines at 100 as noted in format guides, not less.

Reply:
ok. done.